ts/Series/TiledwebMap/TiledwebMapSeries.ts

```diff
@@ -0,0 +1,221 @@
+/* *
+ *
+ *  (c) 2010-2022
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+/* *
+ *  Imports
+ *
+ * */
+
+import type TiledwebMapSeriesOptions from './TiledwebMapSeriesOptions';
+import SeriesRegistry from '../../Core/Series/SeriesRegistry.js';
+
+const {
+    seriesTypes: {
+        map: MapSeries
+    }
+} = SeriesRegistry;
+
+import U from '../../Core/Utilities.js';
+
+const {
+    merge,
+    extend
+} = U;
+
+/* *
+ *
+ *  Declarations
+ *
+ * */
+
+declare module '../../Core/Series/SeriesOptions' {
+    interface SeriesStateHoverOptions {
+
+    }
+}
+
+/**
+ * The series type
+ *
+ * @private
+ * @class
+ * @name Highcharts.seriesTypes.tiledwebmap
+ *
+ * @augments Highcharts.Series
+ */
+class TiledwebMapSeries extends MapSeries {
```

Comment:
Better call it `TiledWebMapSeries` etc. with a capital W, since they are separate words in natural language.